### ISSUE TITLE
"HELP!!!!" and "DOUBLE HELP!!!!" errors in xlights_l4.cpp log

### ISSUE THREAD

**Issue by brett-foy** [xLightsSequencer/xLights#4548]
NOTE: IF YOU DO NOT ATTACH A SEQUENCE THAT DEMONSTRATES YOUR PROBLEM THEN THERE IS A HIGH PROBABILITY YOUR ISSUE WILL JUST BE CLOSED AND NOT INVESTIGATED.

I am trying to help another xlights user diagnose a problem with a sequence that I cannot reproduce.  He states that all other sequences seem to be ok.   I cannot get the problem to re-create on my rig.

**Describe the bug**
When he opens the sequence, after mapping,  when he renders, he gets no render progress bar, and eventually xlights stops responding and he is eventually forced to force shut down and restart xlights.

I am seeing in the log file errors like:

`2024-05-05 05:17:19,231 21460 log_base [WARN] HELP!!!!   Frame #2739 render on model Pixelari Group (6147x4611) layer 3 effect Shader from 108350ms (#2167) to 144550ms (#2890) timed out 10 secs.`
`2024-05-05 05:17:19,887 1392 log_base [WARN] RenderCacheItem::AddFrame failed to allocate frameBuffer.`
`2024-05-05 05:18:19,232 21460 log_base [WARN] DOUBLE HELP!!!!   Frame #2739 render on model Pixelari Group (6147x4611) layer 3 effect Shader from 108350ms (#2167) to 144550ms (#2890) timed out 70 secs.`

`2024-05-05 05:09:57,524 1392 log_base [WARN] RenderCacheItem::AddFrame buffer size changed ... we dont support this.`
^^ this one seems to be happening when rendering a cube model.  The cube models have a lot of Debug entries in the log

We disabled both cubes on his rig, made no difference.  He does not have GPU or any GPU enabled.   I disabled all  GPU in settings on my rig, but still see references to GPU in my logs.  Not sure if this would be related at all.

As stated, I cannot reproduce.

Attaching the xlights debug package - could someone potentially advise if anything jumps out at what could be a likely issue here?  Please advise if .xsq and media would help or if there are potentially some ideas from the logs.

Thanks
[xLightsProblem (1).zip](https://github.com/xLightsSequencer/xLights/files/15217166/xLightsProblem.1.zip)

**Comment by derwin12** [comment]
Check sequence is the usual duplicate node noise. I think you nailed it Gil .. the horizontal buffer fix. Here they have a Mover Group that just "blows up" the render time - never really finished for me. I dropped another prop in there and it works just fine.
[Interesting that the Mac can process a 1 to a bazillion so quick compared to Windows :-) ]
Attached is a 1 effect sequence that doesnt render for me .. can likely get the fix from the CI or nightly tonight.
[bug.zip](https://github.com/xLightsSequencer/xLights/files/15221486/bug.zip)

**Comment by brett-foy** [comment]
Thanks guys.  I was zeroing in on the movers group also, but the strange thing was that it wasn't mapped to in the sequence.  However, there were channel overlaps from a matrix to the moving heads directly.   I figured that could be an issue.    Given what Gil noted on the general # of overall issues in the layout, I advised that a) he should go to .10, and b) fix the channel overlaps specifically with the movers, and c) figure out the render buffer size on the movers all group.
I appreciate you taking a look and for the feedback.   Feel free to close this out, not sure there is anything actionable here, especially given the state of his layout!
